scripts/release.js

```diff
@@ -156,6 +156,19 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install the component app as dependency and build the test app to ensure no errors or build issues
+process.chdir(path.join(__dirname, '..'));
+spawnOrFail('yalc', ['publish']);
+process.chdir(path.join(__dirname, '../..'));
+spawnOrFail('npx', ['create-react-app dependency-check-app']);
+process.chdir(path.join(__dirname, '../../dependency-check-app'));
+spawnOrFail('yalc', ['add amazon-chime-sdk-component-library-react']);
+checkWarning('npm', [`install amazon-chime-sdk-js@1.22.0 styled-components styled-system -q`], null, "amazon-chime-sdk-component-library-react");
+// checkWarning('npm', [`install amazon-chime-sdk-component-library-react@2.2.0 -q &> log.txt`]);
```

Comment:
Is there a reason why we install the 1.22 SDK version?

Reply:
Oh so this commit is only for testing purpose. What I am trying to do here is install the latest version of component library and the earliest version of js sdk and use the method I add in the utilities to catch the warnings. But problem here is if we use yalc instead of npm to install the component as dependency, there will be no warnings for the incompatible peer dependency. So I upload the changes here for others to try and to brainstorm solutions